sdk/openai/Azure.AI.OpenAI/tests/AudioTranscriptionsTests.cs

```diff
@@ -0,0 +1,152 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.AI.OpenAI.Tests;
+
+public class AudioTranscriptionsTest : OpenAITestBase
+{
+    public AudioTranscriptionsTest(bool isAsync)
+        : base(isAsync) // , RecordedTestMode.Live)
+    {
+    }
+
+    [RecordedTest]
+    [TestCase(OpenAIClientServiceTarget.Azure, "json")]
+    [TestCase(OpenAIClientServiceTarget.Azure, null)]
+    [TestCase(OpenAIClientServiceTarget.Azure, "verbose_json")]
+    [TestCase(OpenAIClientServiceTarget.NonAzure, "json")]
+    [TestCase(OpenAIClientServiceTarget.NonAzure, null)]
+    [TestCase(OpenAIClientServiceTarget.NonAzure, "verbose_json")]
```

Comment:
Add test cases for "srt", "vtt" and "text" (where "text" returns `new AudioTranscriptionFormat("text")` in the switch expression below).

Do the same for the `TranslationWorksWithFormat` test below.

Reply:
Good point. I don't think there's any difference in client treatment (they're both just exercising the "AsText" paths) but it's still a great idea to have full coverage. Added and refactored the check a bit.